app/src/main/res/layout/category_card_row_item.xml

```diff
@@ -6,6 +6,14 @@
     android:layout_height="@dimen/category_card_height"
     >
 
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/delete_category_icon"
+        app:layout_anchor="@+id/scoring_category_card_view"
```

Comment:
I think you could change their order, but maybe I'm not familiar with the best practices of android layouts. @jmccormack200 Does it matter at all?